src/core_plugins/kibana/public/discover/components/field_chooser/lib/detail_views/string.html

```diff
@@ -44,6 +44,7 @@ <h5 ng-show="!field.details.error">Quick Count <kbn-info info="Top 5 values base
 
       <a
         ng-href="{{vizLocation(field)}}"
+        ng-if="field.aggregatable"
```

Comment:
I added an additional flag to the field object itself, I think it's a more appropriate place to put that kind of logic. Let me know how that looks to you.